kernel/src/arch/x86_64/rackscale/mem.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2022 University of Colorado and VMware Inc. All Rights Reserved.
```

Comment:
Would it make sense to separate syscall subsystems via directory structure?

E.g., rackscale/file_ops for file operations (fio) and rackscale/process_ops/mem.rs and similar for the process operations?

Reply:
I felt like we can keep it as is for now because it will cause more pain when merging your branch and we renamed/moved all the files here as git isn't that good with tracking these changes. But we should maybe add more structure in the future.

Reply:
That sounds like a good plan!